lib/fp.js

```diff
@@ -174,6 +174,26 @@ const restLeft = fn => (...spreadArgs) => {
   fn(args, ...namedArgs, callback);
 };
 
+// Identity function
+//   x - any argument
```

Comment:
You should use the new documentation comments style, there are plenty of examples of the correct style in this project since all of the comments were updated in https://github.com/metarhia/common/pull/194.

Reply:
How can we describe "any type" or scalar? I thin here we have scalar type